libs/zzz/formula/src/data/util/tag.ts

```diff
@@ -220,6 +220,11 @@ export const own = convert(ownTag, { et: 'own', dst: null })
 export const team = convert(ownTag, { et: 'team', dst: null, ...noName })
 export const target = convert(ownTag, { et: 'target', ...noName })
 export const enemy = convert(enemyTag, { et: 'enemy', dst: null, ...noName })
+team.common.count = team.common.count.withTag({
```

Comment:
Add a comment. Something about avoiding reading from two counts at the same time by accident